templates/project/project.yml

```diff
@@ -121,11 +125,37 @@ Resources:
         PolicyDocument:
           Version: "2012-10-17"
           Statement:
-              - Sid: HostedZoneChangeRecord
+              - Sid: HostedZoneReadRecords
                 Effect: Allow
                 Action:
-                  - route53:*
+                  - route53:Get*
+                  - route53:List*
                 Resource: "*"
+              - Sid: HostedZoneUpdate
+                Effect: Allow
+                Action:
+                  - route53:ChangeResourceRecordSets
+                Resource:
+                  - !Sub arn:${AWS::Partition}:route53:::hostedzone/${ProjectHostedZone}
+
+  ProjectHostedZone:
+    Type: AWS::Route53::HostedZone
+    Condition: DelegateDNS
+    Properties:
+      HostedZoneConfig:
+        Comment: !Sub "Hosted zone for ecs-cli project ${ProjectName}: ${ProjectName}.${ProjectDomainName}"
+      Name: !Sub ${ProjectName}.${ProjectDomainName}
+
+  ProjectDomainDelegationRecordSet:
```

Comment:
Are we going to export this later?

Reply:
No, I don't think so - since it's not going to be imported, necessarily, within this account.